applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -14,7 +14,9 @@
 #include <zephyr/bluetooth/audio/pacs.h>
 #include <zephyr/bluetooth/audio/csip.h>
 #include <zephyr/bluetooth/audio/cap.h>
-#include <../subsys/bluetooth/audio/endpoint.h>
+
+/* TODO: Remove when a get_info function is implemented in host */
```

Comment:
Should have the same comment all places where this file is included

Reply:
Probably yes